.github/workflows/pyright.yml

```diff
@@ -24,23 +24,7 @@ jobs:
 
       - name: Merge CI fixes from sagemath/sage
         run: |
-          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
What does this (not) do?

Reply:
These changes are coming from a different PR, right? In either case, none of the other "Merge CI fixes" steps are using it.